content/source/docs/cloud/api/changelog.html.md

```diff
@@ -21,6 +21,14 @@ Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
     * added [Module List API](./modules.html#list-registry-modules-for-an-organization).
     * updated [Module Delete APIs](./modules.html#delete-a-module).
     * ![cloud][] added public registry module related APIs.
+* ![deprecation] [] The following [Registry Module APIs](./modules.html) have been replaced with newer apis and will be removed in the future.
```

Comment:
Might be worth calling attention to the benefits of the new endpoints -- e.g. better JSON:API compliance, more idiomatic HTTP verbs, etc.

Reply:
The changelog is not supposed to have a lot of detail - mostly just calling out what is changed.